Tensile/Source/client/main.cpp

```diff
@@ -599,31 +692,35 @@ int main(int argc, const char* argv[])
                     {
                         while(listeners.needMoreRunsInSolution())
                         {
-                            auto inputs = dataInit->prepareGPUInputs(problem);
-
-                            auto kernels = solution->solve(problem, *inputs, *hardware);
+                            std::vector<std::shared_ptr<ContractionInputs>> inputs;
+                            std::vector<std::vector<KernelInvocation>> kernels;
+                            for(size_t i = 0; i<flush_count; i++)
+                            {
+                                inputs.push_back(dataInit[i]->prepareGPUInputs(problem));
```

Comment:
I am trying this but initialization is very slow.
Can we call initialization only once and copy initialized data to the remaining buffers?

Reply:
Even if I disabled new parameters, Tensile got stuck at "Generating kernels" with this code change...
It must be something wrong. 
Without the change, it is done in less than a minute.

Reply:
Adding the parameters to validParameters in Common.py as per Majed's suggestion caused the hang. 

Keep in mind these are global parameters, after the fix and moving the flag from your config with the rest of the global parameters with the following syntax seemed to work fine.

FlushMemorySize: 536870912

Reply:
Is the latest version of this changeset still causing slow init and hang problems?